ee/codegen/src/project.ts

```diff
@@ -1118,4 +1152,114 @@ ${errors.slice(0, 3).map((err) => {
       ])
     );
   }
+
+  public getEntrypointId(): string | undefined {
+    const entrypointNode = this.workflowContext.tryGetEntrypointNode();
+    if (isNil(entrypointNode)) {
+      this.workflowContext.addError(
+        new MissingEntrypointIdError(
+          "Cannot determine entrypoint id for the given workflow"
```

Comment:
[nit] we should specify all of these exceptions as `WARNING` unless it's truly fatal (no known examples)